keyboards/scorpius_m32/readme.md

```diff
@@ -0,0 +1,28 @@
+# Scorpius_m32
+
+![Scorpius_m32](https://deskthority.net/wiki/images/e/ed/HHKB_Lite_2.jpg)
+
+This is for the [Scorpius_m32](https://deskthority.net/wiki/HHKB_Lite)
+by PFU, identified by model numbers PD-KB200W/U or PD-KB200B/U. It
+uses the replacement Teensy 2.0 board from [my other
+project](https://github.com/thirteen37/HHKB-Lite-2-Teensy).
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```